extension/test/e2e_visual_test.ts

```diff
@@ -133,6 +133,30 @@ describe('Visual Regression Tests', function () {
     });
   });
 
+  describe('zoom', function () {
+    it('should cycle through zoom levels', async function () {
```

Comment:
Instead of just one test for all zoom levels can we have a single test for each one. Splitting the behaviors will make maintaining the tests a bit more easier in the futuer I think.

example: 'should render at zoom level 1 when 'z' pressed one time' and so forth. Then each test is basically the same except with more key presses. :)